src/support_sphere/lib/presentation/components/checklist_card.dart

```diff
@@ -1,25 +1,36 @@
 import 'package:flutter/material.dart';
 import 'package:intl/intl.dart';
 import 'package:support_sphere/constants/string_catalog.dart';
+import 'package:support_sphere/utils/extensions.dart';
 
 class ChecklistCard extends StatelessWidget {
   final String title;
   final String? frequency;
   final int? stepCount;
   final String? description;
+  final String? notes;
+  final String? priority;
+  final int? completions;
   final bool isInProgress;
   final DateTime? completedDate;
-  final VoidCallback onButtonClicked;
+  final VoidCallback? onButtonClicked;
+  final Widget? trailing;
+  final bool isLEAP;
 
   const ChecklistCard({
     super.key,
     required this.title,
     this.stepCount = 0,
     this.frequency = '',
     this.description = '',
-    this.onButtonClicked = _defaultButtonClicked,
+    this.priority = '',
+    this.completions = 0,
+    this.notes = '',
+    this.onButtonClicked,
     this.isInProgress = false,
     this.completedDate,
+    this.trailing,
+    this.isLEAP = false,
```

Comment:
The "isLeap" prop allows checklist's priority, num of completions, and notes to be displayed on cards.